lms/templates/emails/activation_email.txt

```diff
@@ -0,0 +1,21 @@
+<%! from django.utils.translation import ugettext as _ %>
+${_("You're almost there! Use the link to activate your account to access engaging, high-quality "
+"{platform_name} courses. Note that you will not be able to log back into your account until "
+"you have activated it.").format(platform_name=platform_name)}
+
+% if is_secure:
+https://${ site }/activate/${ key }
+% else:
+http://${ site }/activate/${ key }
+% endif
+
+${_("Enjoy learning with {platform_name}.").format(platform_name=platform_name)}
+
+${_("The {platform_name} Team").format(platform_name=platform_name)}
+
+${_("If you need help, email {support_email}.").format(support_email=support_email)}
```

Comment:
@afzaledx This is missing a "please" :)

> If you need help, **please** email {support_email}.

Also, does `support_email` include the `mailto` link markup that Cloudera requested? If not, we'll need to add it here as well.